pkg/mock/mockcluster/mockcluster.go

```diff
@@ -808,8 +808,8 @@ func (mc *Cluster) MockRegionInfo(regionID uint64, leaderStoreID uint64,
 	followerStoreIDs, learnerStoreIDs []uint64, epoch *metapb.RegionEpoch) *core.RegionInfo {
 	region := &metapb.Region{
 		Id:          regionID,
-		StartKey:    []byte(fmt.Sprintf("%20d", regionID)),
```

Comment:
why change it?

Reply:
In before, regions key will order by [1,2,3,4,5], there are no key range for the buckets. 
In this pr, regions key will order by [10,20,30,40,50], I can put some buckets in one regions.